README.md

```diff
@@ -14,7 +14,7 @@
 [![Latest conda-forge version](https://img.shields.io/conda/vn/conda-forge/jupyterhub?logo=conda-forge)](https://anaconda.org/conda-forge/jupyterhub)
 [![Documentation build status](https://img.shields.io/readthedocs/jupyterhub?logo=read-the-docs)](https://jupyterhub.readthedocs.org/en/latest/)
 [![GitHub Workflow Status - Test](https://img.shields.io/github/workflow/status/jupyterhub/jupyterhub/Test?logo=github&label=tests)](https://github.com/jupyterhub/jupyterhub/actions)
-[![DockerHub build status](https://img.shields.io/docker/build/jupyterhub/jupyterhub?logo=docker&label=build)](https://hub.docker.com/r/jupyterhub/jupyterhub/tags)
+[![Docker Repository on Quay](https://quay.io/repository/jupyterhub/jupyterhub/status "Docker Repository on Quay")](https://quay.io/repository/jupyterhub/jupyterhub)
```

Comment:
### Consistent use of shields.io

Sadly shields.io doesn't provide badges for quay.io, or a "any container registry" badge (already identified by @mathbunnyru :smile: https://github.com/badges/shields/issues/357#issuecomment-1772879156).

If we are to have a badge, I'd love to see it be a shields.io badge as that enables us to provide labels, colors, logos, etc. If there is a YAML/JSON endpoint returning the info we want under some key, we can make a shields.io badge still though.

### Current and PR's badge broken

Before this PR we have a broken badge to report build status from dockerhub, but dockerhub isn't used to build any more - our github release.yaml workflow is.

[![DockerHub build status](https://img.shields.io/docker/build/jupyterhub/jupyterhub?logo=docker&label=build)](https://hub.docker.com/r/jupyterhub/jupyterhub/tags)

What this PR transitions to is also a broken badge.

[![Docker Repository on Quay](https://quay.io/repository/jupyterhub/jupyterhub/status "Docker Repository on Quay")](https://quay.io/repository/jupyterhub/jupyterhub)

### What do we want?

I think it would be nice to have something like the PyPI badge, showing the latest semver version of the main image (quay.iojupyterhub/jupyterhub) - but I don't know how to accomplish that atm. Not using shields.io (preferred) or quay.io directly.

I suggest this badge is dropped for now instead.

Reply:
I've removed it.